src/components/Button/Button.tsx

```diff
@@ -0,0 +1,65 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import "lbh-frontend/lbh/components/lbh-button/_button.scss";
+import "govuk-frontend/govuk/components/button/button.js";
```

Comment:
Importing this isn't enough. We need to actually hook our button up to have the same behaviour. I would expect to see some tests to confirm that it's behvaing as expected, too.